### PR TITLE
CORE-445 Use probe_encrypt

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/pEpEngine.c [hunk 1]
```diff
@@ -4299,16 +4288,19 @@ unlock_keys_with_passphrase(PEP_SESSION session,
 
         if (!current) {
             free_stringlist(*error_accounts);
+            *error_accounts = NULL;
```
Comment (ip-plank-security) [inline src/pEpEngine.c]: I guess this will produce merge conflicts, as the NULLing of error_accounts already has been added with CORE-442, right?

Comment (ghost) [reply src/pEpEngine.c]: Yes, CORE-442. Git tends to be robust about these things. Both (1) same change in the same line and (2) exact same commit in more then one branch -> OK.

Comment (ghost) [reply src/pEpEngine.c]: In any case I will merge them in order, and rebase if necessary. Most probably no problem at all.

Comment (ip-plank-security) [reply src/pEpEngine.c]: then I have no further objections

**Review by ip-plank-security — APPROVED** [review]

**Review by ig-garcia — APPROVED** [review]

**Review by saschabacardit — APPROVED** [review]

**Pull request closed without merging**